docs/src/development/public-ips.md

```diff
@@ -1,254 +1,46 @@
 ---
+# Data: Add and edit regions in `docs/data/regions.yaml`.
+# Styles: See `docs/static/styles/user-customizations.css`.
+# Table shortcode: See `docs/layouts/shortcodes/region_ips.html`.
 title: "Public IP addresses"
 weight: 14
 sidebarTitle: "Public IPs"
 description: |
   Platform.sh regions reach the outside through a limited number of IP addresses.
+  These IP addresses are stable, but not guaranteed to never change.
+  Prior to any future change, all affected customers will receive ample warning.
```

Comment:
I think this information is important, but I don't think  it really works in the description, which should be about making people want to click to read details or whatever.

I'd also switch "all affected customers" so that it's in the second person. Maybe something like "Before any change, you're notified well in advance if you're affected."?